index.html

```diff
@@ -1194,16 +1197,102 @@
+            "width: 3.478em; padding: 0px 0.12em;"><span class="mjx-numerator"
+            style="width: 3.478em; top: -1.447em;"><span id="MJXc-Node-111"
+            class="mjx-msub"><span class="mjx-base" style=
+            "margin-right: -0.003em;"><span id="MJXc-Node-112" class=
+            "mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em; padding-right: 0.003em;">d</span></span></span><span class="mjx-sub"
+            style=
+            "font-size: 70.7%; vertical-align: -0.219em; padding-right: 0.071em;"><span id="MJXc-Node-113"
+            class="mjx-mrow" style=""><span id="MJXc-Node-114" class=
+            "mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+            "padding-top: 0.372em; padding-bottom: 0.372em;">16</span></span><span id="MJXc-Node-115"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-116"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-117"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span></span><span class="mjx-denominator"
+            style="width: 3.478em; bottom: -0.778em;"><span id="MJXc-Node-118"
+            class="mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+            "padding-top: 0.372em; padding-bottom: 0.372em;">65535.0</span></span></span><span style="border-bottom: 1.3px solid; top: -0.296em; width: 3.478em;"
+            class="mjx-line"></span></span><span style=
+            "height: 2.225em; vertical-align: -0.778em;" class=
+            "mjx-vsize"></span></span></span></span></span></span></span>
+          </p>
+          <h3>
+            Read the data from WebGL texture
+          </h3>
+          <p>
+            We list here some of the possible approaches:In addition to , there
+            are two ways available for asynchronous access:
+          </p>
+          <ul>
+            <li>Synchronous readPixels requires the least amount of code and it
+            is available om WebGL1. See the <a>readPixels from float</a>
+            example for details.
+            </li>
+            <li>Asynchronous readPixels using pixel buffer objects should
+            remove the blocking while waiting for the readPixels call.
+            </li>
+            <li>Transform feedback with GetBufferSubData(Async) provides
+            synchronos and asynchronous read access to depth and color texture
+            data processed in vertex shader.
+            </li>
+          </ul>
+          <p></p>
+          <p class="note">
+            Performance of synchronous <a>readPixels from float</a> example in
```

Comment:
a bit hard to read.

The "readPixel from float" example above, while being simplistic, suffers from some performance issues, due to the fact that ...  This can be mitigated by...